modules/apps/data-engine/data-engine-test/src/testFunctional/tests/documentsadministration/fields/DMRichTextField.testcase

```diff
@@ -107,12 +107,37 @@ definition {
 	}
 
 	@description = "This is a test for LPS-98076. As a Developer, I want to Implement the HTML field type"
-	@ignore = "true"
 	@priority = "4"
 	test SetRichTextSearchableAsByKeyword {
```

Comment:
It is not possible to set Searchable property for Rich Text as Keyword.

The available options are: Disable or Text.

Please, change the test method to: SetRichTextSearchableAsBy**Text**